services/src/main/java/com/linkedin/kmf/services/configs/TopicRebalancerConfig.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.services.configs;
+
+import java.util.Map;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import static org.apache.kafka.common.config.ConfigDef.Range.atLeast;
+
+public class TopicRebalancerConfig extends AbstractConfig {
+
+  private static final ConfigDef CONFIG;
+
+  public static final String ZOOKEEPER_CONNECT_CONFIG = CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG;
+  public static final String ZOOKEEPER_CONNECT_DOC = CommonServiceConfig.ZOOKEEPER_CONNECT_DOC;
+
+  public static final String TOPIC_CONFIG = CommonServiceConfig.TOPIC_CONFIG;
+  public static final String TOPIC_DOC = CommonServiceConfig.TOPIC_DOC;
+
+  public static final String REBALANCE_EXPECTED_RATIO_CONFIG = "rebalance.expectedPartitionBrokerRatio";
+  public static final String REBALANCE_EXPECTED_RATIO_DOC = "The expected ratio of partition / broker.  When the actual ratio falls below this new partitions are created.";
+
+  public static final String REBALANCE_INTERVAL_MS_CONFIG = "rebalance.intervalMs";
```

Comment:
To stay inline with the config naming conversion in Kafka, this should probably be named rebalance.interval.ms.